README.rst

```diff
@@ -47,8 +51,14 @@ Follow the usual CMake build workflow:
     
     PKG_CONFIG_PATH=FOLDER_WITH_LIBXC_PC_FILES FC=gfortran cmake [...]
 
+  or::
+
+    CMAKE_PREFIX_PATH=YOUR_LIBXC_INSTALL_FOLDER FC=ifort cmake [...]
+    
+    PKG_CONFIG_PATH=FOLDER_WITH_LIBXC_PC_FILES FC=ifort cmake [...]
```

Comment:
Again, this is superfluous. Also, the `CMAKE_PREFIX_PATH` example will only work if we upgrade to recent libxc, so that belongs into the other PR.